template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/impala/ImpalaVolumeConfigProvider.java

```diff
@@ -74,4 +95,8 @@ public Set<String> getRoleTypes() {
     public boolean sharedRoleType(String roleType) {
         return false;
     }
+
+    private boolean checkTemporaryStorage(HostgroupView hostGroupView, Integer temporaryStorageVolumeCount) {
+        return hostGroupView != null && hostGroupView.getTemporaryStorage() == TemporaryStorage.EPHEMERAL_VOLUMES && temporaryStorageVolumeCount != 0;
```

Comment:
TemporaryStorage.EPHEMERAL_VOLUMES vs TemporaryStorage.EPHEMERAL_VOLUMES_ONLY <--- Is the specific check in place because the mount paths are different when a node has EPHEMERAL_VOLUMES_ONLY vs EPHEMERAL_VOLUMES ?, and the EPHEMERAL_VOLUMES_ONLY scenario is handled the same as the general approach?

Reply:
cc @nadamdb - may have a better sense of how EPHEMERAL_VOLUMES_ONLY vs EPHEMERAL_VOLUMES works.